src/AppBundle/Service/Taxation/FloatCalculator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\Service\Taxation;
+
+use Sylius\Component\Taxation\Calculator\CalculatorInterface;
+use Sylius\Component\Taxation\Model\TaxRateInterface;
+
+final class FloatCalculator implements CalculatorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function calculate(float $base, TaxRateInterface $rate): float
+    {
+        if ($rate->isIncludedInPrice()) {
```

Comment:
Maybe add a comment ?
`if the base is all taxes included`

Reply:
This is already documented in `TaxRateInterface`